pkg/common/config/config.go

```diff
@@ -67,21 +67,17 @@ var keyToSecretMappingMutex = sync.Mutex{}
 
 // Upgrade config keys.
 var Upgrade = struct {
-	// UpgradeToCISIfPossible will upgrade to the most recent cluster image set if it's newer than the install version
-	// Env: UPGRADE_TO_CIS_IF_POSSIBLE
-	UpgradeToCISIfPossible string
+	// UpgradeToLatest will look for the newest-possible version and select that
+	// Env: UPGRADE_TO_LATEST
+	UpgradeToLatest string
 
-	// OnlyUpgradeToZReleases will restrict upgrades to selecting Z releases on stage/prod.
-	// Env: ONLY_UPGRADE_TO_Z_RELEASES
-	OnlyUpgradeToZReleases string
+	// UpgradeToNextY will look for the next Y version for the cluster and select that
```

Comment:
`UpgradeToNextY` and `upgrade-to-next-y` definitions are confusing.. Will this upgrade to the latest .z of the latest .y? If so this should be title `UpgradeToLatestY` and defined as "will look for the latest Y version and update to its latest Z"

Reply:
Here was my thought behind it....
"Latest Y" could mean, if (In Integration) we installed 4.5, and a 4.7 nightly is available, it would imply it could upgrade to 4.7. Next implies Y+1

The reality is, it's only ever looking at the available upgrade edges so that case is _extremely_ unlikely to happen and it would only ever do a Y+1

All that said, if you think it should be s/next/latest/ I can crank that out quick. :)